### ISSUE TITLE
Fullscreen windows under panels and on-top windows.

### ISSUE THREAD

**Issue by uberRegenbogen** [linuxmint/mint20-beta#190]
If any window is set to on-top, fullscreen windows (e.g. terminals, mpv, VLC) appear under them, and panels.  If there are no on-top windows, everything is fine.

Observed in LM 19.3 Cinnamon, as well as LM 20 beta.  I do not recall this occuring in LM 17.3 Cinnamon.

**Comment by xenopeek** [comment]
Please add a screenshot to illustrate the issue.

**Comment by xenopeek** [comment]
What I found is that if a window has Always on Top toggled that window **and** the panel will stay on top of any other window that goes fullscreen. That the window set to Always on Top stays on top should be expected I think but that the panel also stays on top not.

Is that what you meant? If not please add a screenshot to better illustrate your issue.

Steps to reproduce:

1. Open some program e.g. Calculator.
2. Right-click its titlebar and toggle Always on Top.
3. Open some other program, that has the fullscreen option, e.g. open a picture file with Image Viewer.
4. In Image Viewer press F11 to toggle fullscreen.
5. Image Viewer goes fullscreen but the panel stays visible and overlaps it.

See below on the left Image Viewer in fullscreen mode with Calculator set Always on Top and to the right the same without Always on Top:
![image](https://user-images.githubusercontent.com/21159972/85570789-f1878800-b633-11ea-8df3-2ea8a4cc15f5.png)

**Comment by uberRegenbogen** [comment]
Yes.  Though i'm thinking that fullscreen windows should obscure on-top windows, as well.  (I'm pretty sure that they used to.)

**Comment by clefebvre** [comment]
ah... that's a good question, whether they should or not.

My first reaction was to agree. Yes, absolutely, fullscreen windows should cover it all... and I'm pretty sure they did in the past also.

But can it be handy to be able to overlay a window on top of a fullscreen app? Yes it can, absolutely but the panel shouldn't.

Are there cases where you'd want a window to be on top... but also need a window to be fullscreen on the same monitor? I can't think of any.

Food for thoughts. I'm moving this to cinamon.
